picard/util/checkupdate.py

```diff
@@ -144,12 +144,15 @@ def _display_results(self):
                 webbrowser2.open(self._available_versions[key]['urls']['download'])
         else:
             if self._show_always:
+                # TODO: Remove temporary workaround once https://github.com/python-babel/babel/issues/415 has been resolved.
+                babel_415_workaround = PROGRAM_UPDATE_LEVELS[self._update_level]['title'] if self._update_level in PROGRAM_UPDATE_LEVELS else 'unknown'
```

Comment:
Currently this will loose us the string `'unknown'`, we have to mark it as extractable by using `N_('unknown')`

Actually this is the one place here where I would not treat this as a workaround but actually clean it up. This one line conditional with deep variable access is a bit awkward to read. I personally would clean it up to be:

```
if self._update_level in PROGRAM_UPDATE_LEVELS:
    update_level = PROGRAM_UPDATE_LEVELS[self._update_level]['title']
else:
    update_level = N_('unknown')
```

Way easier to read IMHO

Reply:
Thanks for noticing this, i missed it :(